planetary-social/nos
# updated color assets for and code in ActionButton.swift

## Issues covered
https://github.com/planetary-social/nos/issues/1375

## Description
- Updated Colors.xcassets to add needed semantically labeled color assets
- Updated related code in ActionButton.swift to use semantically labeled color assets

## How to test
- compare existing non-semantically-color values to new values in Colors.xcassets and verify they are the same
- visually check the preview of ActionButton.swift before and after the described changes

Note that existing unit tests passed. 

## Screenshots/Video
ActionButton preview before changes:
![ActionButtonBefore](https://github.com/user-attachments/assets/d42ab698-fd3a-45ae-8c9a-25385e5f1944)
ActionButton preview after changes:
![ActionButtonAfter](https://github.com/user-attachments/assets/15e24ce9-5ee7-4e23-bc90-72db87880eca)

## Notes
Not sure how you wish to proceed with this ticket; I feel like approving changes a few files at a time going forward and keeping those changes on a non-main branch in the main repo would allow these changes to be reviewed more easily and allow for a squash commit to merge them in to main. Please advise.
